### PR TITLE
Update job chart version

### PR DESCRIPTION
**Pull request by vijayrajagopalan-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-2645

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by nitishsingla1234 — DISMISSED** [review]

**Review by AdilOoze — APPROVED** [review]

File: charts/rd-judicial-data-load/Chart.yaml [hunk 1]
```diff
@@ -3,12 +3,12 @@ appVersion: "0.1"
 description:  Data load from Judicial HR systems
 name: rd-judicial-data-load
 home: https://github.com/hmcts/rd-judicial-data-load
-version: 0.1.9
+version: 0.1.10
 maintainers:
```
Comment (shashank-rastogi) [inline charts/rd-judicial-data-load/Chart.yaml:7]: can we not just move to version 0.2 instead of 0.1.10?

Comment (vijayrajagopalan-hmcts) [reply charts/rd-judicial-data-load/Chart.yaml:7]: Makes sense. Have updated.

File: charts/rd-judicial-data-load/Chart.yaml [hunk 2]
```diff
@@ -3,12 +3,12 @@ appVersion: "0.1"
 description:  Data load from Judicial HR systems
 name: rd-judicial-data-load
 home: https://github.com/hmcts/rd-judicial-data-load
-version: 0.1.9
+version: 0.2.0
```
Comment (nitishsingla1234) [inline charts/rd-judicial-data-load/Chart.yaml:6]: perhaps we can make it 2.01 for next iteration.

**Review by nitishsingla1234 — APPROVED** [review]

**Review by SujeethKumarAri — APPROVED** [review]

File: charts/rd-judicial-data-load/Chart.yaml (hunk 2 above)
Comment (vijayrajagopalan-hmcts) [reply charts/rd-judicial-data-load/Chart.yaml:6]: Yes

**Pull request merged**